src/Microsoft.HttpRepl/Program.cs

```diff
@@ -127,7 +130,9 @@ private static HttpClient GetHttpClientWithPreferences(IPreferences preferences)
         {
             if (preferences.GetBoolValue(WellKnownPreference.UseDefaultCredentials))
             {
+#pragma warning disable CA2000 // Dispose objects before losing scope
                 return new HttpClient(new HttpClientHandler { UseDefaultCredentials = true });
```

Comment:
Who owns disposal of the HttpClient?  How often are we creating new instances?

Reply:
There should be exactly one `HttpClient` for the lifetime of the application. We don't dispose it directly, but we'd only do so on shutdown of the application anyway. For the record, that CA2000 error was for HttpClientHandler, not HttpClient.